MeowBili/MeowBiliApp.swift

```diff
@@ -469,3 +472,12 @@ public func updateBuvid() {
         }
     }
 }
+
+
+func currentGlobalSystemTime() -> String {
```

Comment:
```suggestion
func _currentGlobalSystemTime() -> String {
```

为与App运行无关，仅用于调试的函数命名前添加前缀`_`。

Reply:
草，还有这种规定吗